### DISCUSSION TITLE
Availability EBus adapter

### DISCUSSION THREAD

**Discussion by MoosJ80** [john30/ebusd#761] [General]
Hello,

Is the EBus adapter available for shipping at the moment?

**Comment by john30** [comment]
see #554

**Comment by jannis-h** [comment]
I have a v3.0 with WiFi available that I don't need anymore, in case you are still interested: [https://forum.fhem.de/index.php?topic=132375.0](https://forum.fhem.de/index.php?topic=132375.0)
